### DISCUSSION TITLE
Don't show disambiguation when matches are identical

### DISCUSSION THREAD

**Discussion by galantra** [espanso/espanso#1745] [Feature requests and Ideas]
![image](https://github.com/espanso/espanso/assets/52420923/2047dbde-8967-4ef2-8f05-41ec6f11ff3e)

In such a case, where they are exact duplicates, espanso should simply execute one of them as if there was only one match.

**Comment by smeech** [comment]
This might be useful as an option, especially where installed packages introduce duplications, but you'd still be left with some which result in the same output but via different coding routes.

I rely on the current behaviour to identify duplicates in my own expansions, so I can remove them. I only wish the search trigger window would be more helpful in identifying _where_ a specific `trigger:` can be found, e.g. by showing details on right-click.
